src/Export/UsersExport.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace EscolaLms\CsvUsers\Export;
+
+use EscolaLms\Auth\Enums\GenderType;
+use Illuminate\Contracts\Auth\Authenticatable;
+use Illuminate\Support\Collection;
+use Maatwebsite\Excel\Concerns\FromCollection;
+use Maatwebsite\Excel\Concerns\WithHeadings;
+
+class UsersExport implements FromCollection, WithHeadings
+{
+    private Collection $users;
+
+    public function __construct(Collection $users)
+    {
+        $this->users = $users;
+    }
+
+    public function collection(): Collection
```

Comment:
this is fine 

but ....

we have a way to extends users so you should be able to get all the fields here 

you should us this https://github.com/EscolaLMS/Auth/blob/main/src/Http/Resources/UserResource.php 

```php
UserResource::collection($this->users) 
```

should work 

ps. this is how our extension works https://github.com/EscolaLMS/Courses/blob/main/src/EscolaLmsCourseServiceProvider.php#L91 

@pa-cholek please comment

Reply:
I don't think that UserResource::collection will work as it creates a JsonResourceCollection and not a normal Collection... and I'm not even sure why you would want to use UserResource for export/import, we probably want to export/import all user fields and not only those visible in api responses?